### PR TITLE
Fairmat 2024: additional fields in NXmonochromator

### PR DESCRIPTION
**Pull request by lukaspie**

### CODE REVIEW

File: base_classes/NXmonochromator.nxdl.xml [hunk 1]
```diff
@@ -60,6 +60,26 @@
     <field name="energy_errors" type="NX_FLOAT" units="NX_ENERGY">
         <doc>energy standard deviation</doc>
     </field>
+    <field name="energy_dispersion" type="NX_FLOAT" units="NX_ANY">
+        <doc>
+             Energy dispersion at the exit slit, typically given as eV/mm.
```
Comment (sanbrock) [inline base_classes/NXmonochromator.nxdl.xml:65]: this can now be specified

File: base_classes/NXmonochromator.nxdl.xml [hunk 2]
```diff
@@ -60,6 +60,26 @@
...
+        </doc>
+    </field>
+    <field name="wavelength_dispersion" type="NX_FLOAT" units="NX_ANY">
+        <doc>
+             Wavelength dispersion at the exit slit.
```
Comment (sanbrock) [inline base_classes/NXmonochromator.nxdl.xml:70]: units can be specified

**Comment by prjemian** [comment]
But the Schema will need to be updated for this most recent NIAC decision.
So start using it, knowing that soon the Schema will accept it.

**Comment by phyy-nx** [comment]
Approved by NIAC vote, given the unit fixes listed above, in [NIAC 2024](https://www.nexusformat.org/NIAC2024.html)

File: base_classes/NXmonochromator.nxdl.xml (hunk 1 above)
Comment (lukaspie) [reply base_classes/NXmonochromator.nxdl.xml:65]: done

File: base_classes/NXmonochromator.nxdl.xml (hunk 2 above)
Comment (lukaspie) [reply base_classes/NXmonochromator.nxdl.xml:70]: done

**Comment by lukaspie** [comment]
@sanbrock ready for review before NIAC member can review.

**Review by PeterC-DLS — APPROVED** [review]
LGTM

**Pull request merged**